server/src/main/java/com/flipkart/varadhi/auth/AuthenticationOptions.java

```diff
@@ -32,7 +32,8 @@ public <T> T asConfig(Class<T> configClass) {
     }
 
     public enum Mechanism {
-        jwt
+        jwt,
+        test
```

Comment:
added test mechanism for e2e tests to work with authn and authz enabled.
allows to pass the user via X-Auth-Token header